RecoTracker/LSTCore/src/alpaka/Kernels.h

```diff
@@ -390,15 +390,15 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
           }
           const int minNHitsForDup_pLS = 3;
           if (npMatched >= minNHitsForDup_pLS) {
-            rmPixelSegmentFromMemory(segmentsPixel, idxToRemove, secondpass);
+            rmPixelSegmentFromMemory(pixelSegments, idxToRemove, secondpass);
           }
           if (secondpass) {
             float dEta = alpaka::math::abs(acc, eta_pix1 - eta_pix2);
             float dPhi = calculate_dPhi(phi_pix1, phi_pix2);
```

Comment:
uhm, I've updated the target/master branch, but this line still has `calculate_dPhi`; odd.

Reply:
I would have to rebase for it to change, wouldn't I?

Reply:
well, yes and no. you had only one non-merge commit, it should still properly detect the diff.
True, if you rebase it wouldn't have a way to confuse.